Procurement/Controls/AbstractStashTabControl.cs

```diff
@@ -106,12 +106,34 @@ public void Control_Loaded(object sender, RoutedEventArgs e)
 
         protected void SetBackground(Grid childGrid, Item item)
         {
-            if (item is Gear && (item as Gear).Rarity != Rarity.Normal && (item as Gear).Explicitmods == null)
+            if (item is Gear == false)
```

Comment:
var gear = item as Gear
if(gear == null)
    return;

save's boxing/unboxing it twice.

Reply:
I tend to default to 'is' to avoid unnecessary casting, but in this case it should always be gear so saving the boxing/unboxes makes sense as it will have to cast right after anyway.